### DISCUSSION TITLE
em Lisboa

### DISCUSSION THREAD

**Discussion by jnassula** [anacsv/tech-br-em-pt#9] [Alimentação]
- **Nome:** Delirium Café Bar

- **Tipo:** Pub

- **Onde fica:** Calçada Nova de São Francisco 2A, 1200-289 Lisboa

- **O que vc comeu e gostou:** Muita variedade em cervejas, ótimo lugar para tomar cervejas mais fortes que não se encontra em qualquer lugar.

Muitas pessoas conhecem a cerveja Delirium, mas não associam o nome a marca.
A Delirium é a cerveja do elefante cor-de-rosa. Segue uma imagem abaixo para melhor associação ;)
![Cerveja Delirium](https://media-cdn.tripadvisor.com/media/photo-s/17/2f/d5/06/delirium-tremens-um-icone.jpg)

O bar também fica um pouco escondido, mas ao ver essa imagem, não tem como errar.
![Delirium Café Bar](https://images.squarespace-cdn.com/content/v1/560dbc25e4b0969564758eb5/1553083344183-OZKKZW9NM2ZJ4QK38OUQ/lisbon-portugal-stairs.jpg)

**Comment by anacsv** [comment]
Migo do céu, provei essa cerveja ontem aqui no mercado bom sucesso e hoje estou morrendo de ressaca, isso que foi só um copo hahaha

**Reply by jnassula** [reply]
Cerveja forte é boa pq não precisa beber muitas, logo...
Não gasta tanto hahaha

**Reply by jnassula** [reply]
A última vez que fui para o Porto, fiquei hospedado ao lado desse mercado

**Reply by anacsv** [reply]
Sim tem vantagens pra quem não é miojo igual eu (cozido em 3 min)
